vsphere_plugin_common/__init__.py

```diff
@@ -1485,6 +1490,13 @@ def _update_vm(self, server, cdrom_image=None, remove_networks=False):
                     'Edit cdrom from template. '
                     'Template should have no inserted cdroms.')
                 cdrom_attached = True
+                # skip if cdrom is already attached
+                if isinstance(
+                    device.backing, vim.vm.device.VirtualCdrom.IsoBackingInfo
+                ):
+                    if str(device.backing.fileName) == str(cdrom_image):
+                        logger().warn("Cdrom is already attached")
```

Comment:
Should this be a warning or informational? It seems like maybe it's informational because it's not a situation that should be considered negatively at all?

Reply:
fixed in d72904d0d333330a9f7841ad8e5b5264460c821e